odl/contrib/fom/test/test_supervised.py

```diff
@@ -124,6 +124,53 @@ def test_psnr(space):
     assert result == pytest.approx(expected)
 
 
+def test_ssim(space):
+    ground_truth = odl.phantom.white_noise(space)
+
+    # SSIM of true image should be either
+    # * 1 with force_lower_is_better == False,
+    # * -1 with force_lower_is_better == True and normalized == False,
+    # * 0 with force_lower_is_better == True and normalized == True.
+    result = odl.contrib.fom.ssim(ground_truth, ground_truth)
+    result_normalized = odl.contrib.fom.ssim(ground_truth, ground_truth,
+                                             normalized=True)
+    result_flib = odl.contrib.fom.ssim(ground_truth, ground_truth,
+                                       force_lower_is_better=True)
+    result_nf = odl.contrib.fom.ssim(ground_truth, ground_truth,
+                                     normalized=True,
+                                     force_lower_is_better=True)
+    assert pytest.approx(result) == 1
```

Comment:
Usually we write these the other way around (no need to fix now), but language wise it reads nicer as "result is approximately 1", instead of "approximately result is 1"